### ISSUE TITLE
Make Summary tool screen readable

### ISSUE THREAD

**Issue by sielmm** [sgsinclair/Voyant#407]
The Summary tool is almost entirely text and so it should be straightforward to make it accessible. The text must be made available to keyboard access and screen readers. Currently it has ARIA `role=Ignored` which is probably a good place to start.
